pkg/kademlia/kademlia.go

```diff
@@ -193,6 +193,14 @@ func (k *Kademlia) Bootstrap(ctx context.Context) error {
 	if len(k.bootstrapNodes) == 0 {
 		return BootstrapErr.New("no bootstrap nodes provided")
 	}
+
+	for _, node := range k.bootstrapNodes {
+		_, err := k.nodeClient.Ping(ctx, node)
+		if err != nil {
+			return err
```

Comment:
is it really an error if we fail to ping _any_ of the bootstrap nodes? how about returning an error only if we fail to ping _all_ of them

Reply:
sounds good